### DISCUSSION TITLE
Day of Week not showing when using date-time picker

### DISCUSSION THREAD

**Discussion by MacelinCane** [mdbootstrap/TW-Elements#2048] [Support from community] [answered]
Added 
```data-te-format="dddd, dd, mmm, yyyy"``` 
to element. 

Expected result to include the long day of the week, followed by the date in dd/mm/yyyy, then the time.

Got the default format. 
The long day of the week never shows. Further, if I re-order the format, to
```data-te-format="dddd, mmm, dd, yyyy"```
it doesn't honor that either.  The output is always day, month, year as xx/xx/xxxx (followed by the time in 12hr)

Instructions followed are here: https://tw-elements.com/docs/standard/forms/datepicker/
Video showing code and behavior here: https://www.screencast.com/t/gOvOkz4RDw5

I have the initialization all working as expected as well, but it seems additional js is not needed for this, but maybe the docs aren't clear enough and that's not true?

What am I missing?

**Comment by juujisai** [comment, marked as answer]
Hi! Try initializing the component via JS and providing the options for datepicker here. It should be working properly like that:

```
const dateTimepickerContainer = document.querySelector("#datetimepicker");

new Datetimepicker(dateTimepickerContainer, {
  datepicker: {
    format: "dddd, dd, mmm, yyyy",
  },
});
```

Make sure to remove the `data-te-date-timepicker-init` data attribute or `Datetimepicker` from the `initTE` method so that the component wont initialize twice.

**Comment by NickSmithTech** [comment]
By the way, this solution worked.

Thanks!
